test/prod/RToken.test.ts

```diff
@@ -1037,15 +1037,15 @@ describe('RToken contract', function () {
         .withArgs(basketTokens.length, newTokens.length)
 
       // Check no balance in contract
-      expect(await bskToken.balanceOf(rToken.address)).to.equal(bn(0))
-      expect(await bskToken.balanceOf(addr1.address)).to.equal(bn(0))
-      expect(await bskToken.balanceOf(addr2.address)).to.equal(bn(0))
+      expect(await bskToken.balanceOf(rToken.address)).to.equal(bn('0'))
```

Comment:
For example (lots of changes like this)...

```suggestion
      expect(await bskToken.balanceOf(rToken.address)).to.equal(0)
```